.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+name: CLEO 5 Test Build
+
+on:
+  push:
+    paths-ignore:
+    - ".github/*"
+    - "*.md"
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1
+
+      - name: Build Projects
+        shell: cmd
+        run: |
+          set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
+          msbuild -m CLEO5.sln /property:Configuration=Release /property:Platform=GTASA
+          msbuild -m cleo_plugins/CLEO_Plugins.sln /property:Configuration=Release /property:Platform=x86
+
+      - name: Prepare Files
+        id: prepare_archive
+        shell: cmd
+        run: |
+          @REM create output directory
+          mkdir .output\Release\cleo
+          mkdir .output\Release\cleo\cleo_plugins
+
+          @REM copy files
+          copy third-party\bass\bass.dll .output\Release\bass.dll
+          copy source\cleo_config.ini .output\Release\cleo\.cleo_config.ini
+          copy cleo_plugins\.output\*.cleo .output\Release\cleo\cleo_plugins
+          copy cleo_plugins\.output\*.ini .output\Release\cleo\cleo_plugins
+
+      - uses: actions/upload-artifact@v4
+        with:
+          compression-level: 0
```

Comment:
No compression at all?

Reply:
> No compression at all?

You can change it by 0-9, i left it for you, 6 is default.